workflow.md

```diff
@@ -0,0 +1,41 @@
+## Helper bash script to automatically apply the patches
```

Comment:
I've described the process in https://github.com/conda-forge/python-feedstock/pull/481/files#diff-6c00e531f1a7c5082ca5f03b2c5d748a7c5b0216ff5120615f6c24e2dbe44460

Reply:
oh great. I clearly didn't read the readme closely enough. i'll delete this workflow.md file

Reply:
It's not there yet. Only in the 3.10 branch of my fork.